### PR TITLE
Feature/216 add a field to provide a link to application form pack

### PR DESCRIPTION
**Pull request by despo**

### CODE REVIEW

**Comment by despo** [comment]
[Add a field to provide a link to application form/pack/whatever to enable interested applicants to take next step](https://trello.com/c/pmaUCR4m/216-add-a-field-to-provide-a-link-to-application-form-pack-whatever-to-enable-interested-applicants-to-take-next-step)

File: config/locales/en.yml [hunk 1]
```diff
@@ -96,3 +98,7 @@ en:
     save_and_continue: 'Save and continue'
     find: 'Find'
     delete: 'Delete'
+
+  errors:
```
Comment (tahb) [inline config/locales/en.yml]: This should probably live in in the activerecord.yml file.

File: app/views/hiring_staff/vacancies/application_details/new.html.haml [hunk 2]
```diff
@@ -16,6 +16,12 @@
                 as: :email,
                 required: true
 
+      = f.input :application_link,
```
Comment (tahb) [inline app/views/hiring_staff/vacancies/application_details/new.html.haml]: I feel like a hint would work well here as I can imagine `Application URL` will not be clearly understood by all.

Comment (despo) [reply app/views/hiring_staff/vacancies/application_details/new.html.haml]: fixed, it was mapped to the wrong locale as I copy pasted the gds fields below and edited 🤔

**Pull request merged**